assets/vue/components/migrate-notice.vue

```diff
@@ -1,27 +1,27 @@
 <template>
-	<div class="migrate-notice" v-if="Object.keys(this.migrationData).length && !dismissed">
+	<div class="migrate-notice" v-if="Object.keys(this.migration_data).length && !dismissed">
```

Comment:
You're already checking the migration data property when sending it to the component inside `main.vue`. You can  check the prop here - with `v-if="migration_data"` - instead of checking in both places.